src/constants.ts

```diff
@@ -68,3 +68,6 @@ export const ID_TOKEN_ERRORS = [
   "NONCE_NOT_MATCHING",
   "INCORRECT_VOT",
 ];
+
+export const INVALID_ISSUER = "https://example.com/identity-provider";
+export const oneDayInSeconds = 86400;
```

Comment:
nitpick but can we make this consistent casing?

Reply:
Yep have fixed!